### DISCUSSION TITLE
New Maintainers Announcement

### DISCUSSION THREAD

**Discussion by coreydaley** [gorilla/.github#9] [Announcements]
We were going to wait until the transition was complete to make an announcement, but it seems that the cat is out of the bag…

We are excited to confirm that the Gorilla web toolkit project has a new group of Core Maintainers!

We are all thrilled by the positive response from the community so far. Here is a list of what we are currently working on to complete the transition process:

- Updating the project to support the latest Go version
- Implementing GitHub features such as Branch Protection, GitHub Actions, and more.
- Updating documentation for contributing, code of conduct, reporting vulnerabilities, etc.

To address many comments that we have seen - we would like to clarify that Red Hat is not taking over this project. While the new Core Maintainers all happen to work at Red Hat, our hope is that developers from many different organizations and backgrounds will join the project over time.

We are hoping to have the transition completed by the end of July, 2023. You can follow our progress by watching for updates at the following locations:

- https://gorilla.github.io/blog
- https://github.com/gorilla
- https://www.reddit.com/r/golang/
- https://groups.google.com/g/gorilla-web

Thank you for your support and we look forward to collaborating with the community!

**Comment by qianbeibuzui** [comment]
that's great

**Comment by painhardcore** [comment]
Is there something specific you need assistance with right now?

**Comment by here-Leslie-Lau** [comment]
That's cool!

**Comment by iamonah** [comment]
this is fantastic and I hope it ends up not becoming a red hat thing.

**Comment by andre-dasilva** [comment]
thats great. thanks for keeping those amazing libraries alive

**Comment by avecost** [comment]
Nice! Glad to hear this news!

**Comment by ch3nnn** [comment]
Glad to hear this news!

**Comment by sjakk** [comment]
nice

**Comment by theghostmac** [comment]
Awesome! I was surprised to see my code editor not throwing archive warnings this morning. LFG 🚀🚀

**Comment by gnoe** [comment]
This is one of the best news I've read today! Thank you so much for keeping alive this amazing projects! 🚀 🌔

**Comment by indyarocks** [comment]
Nice 🚀 🎸

**Comment by jerodsanto** [comment]
We'd love to have the new maintainers join us for an episode of the [Go Time podcast](https://gotime.fm) if y'all are interested! 💚

**Reply by donuts-are-good** [reply]
Thanks for the show! I appreciate the different types of people you find to interview. The ones with devs from outside of SV or even USA are some of the best and most relatable episodes. Keep up the good work

**Comment by titpetric** [comment]
I'd like to join the maintainers to be able to address some of the reported issues / PRs as time allows.
Is there some rite of passage to invoke (CLA), or who do I make the request to? Is there interest to transfer the org?

@coreydaley

**Reply by coreydaley** [reply]
The best way would be to just start picking up issues and submitting pull requests. I'm actually not sure who is in charge of this org now, let me poke around and see.

**Reply by jaitaiwan** [reply]
It's @AlexVulaj @apoorvajagtap and @bharat-rajani.
